static/sass/6-components/_footer.scss

```diff
@@ -1,6 +1,14 @@
 .c-site-footer {
-  .border--yellow_notch {
-    border-top: 2px solid $color-yellow-tribune;
-    margin: $size-xs 0;
+  
+  &__logo {
+    font-size: 4rem;
+    margin-bottom: 2.6rem;
+  }
+  &__icon {
```

Comment:
I'm torn here. Part of me thinks social icons in the footer will be common enough that the style for it should live at the DS level. Part of me thinks it'll make more sense at the repo level given, for example, the UMP footer doesn't have social icons. IDK.

Reply:
I'm torn too! That's a tough one. I like the idea of it being in the DS because that'd be the last remaining style, allowing it to only pull from the DS with no repo-specific code. Same case with the /texastribune footer. I don't like UMP getting unused CSS, but it _is_ only 1 declaration 🤔

Reply:
Up to you, but my final vote is for adding to DS. It wouldn't shock me at all if someone asks me to add the social icons to the UMP footer, in which case there will be no unused DS code!